src/Util/Crypto.php

```diff
@@ -30,9 +30,9 @@ public static function rsaSign($data, $key, $alg = 'sha256')
 
 	/**
 	 * @param string $data data
-	 * @param string $sig  signature
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.